doc/source/tune.rst

```diff
@@ -0,0 +1,160 @@
+Ray.tune: Efficient distributed hyperparameter search with Ray
```

Comment:
I think the title is way too long - not very user friendly from the sidebar

Reply:
Perhaps the following: `Ray.tune: Distributed hyperparameter search`

Reply:
Changed (but I kept "efficient")